src/common/_modules/apim/networking.tf

```diff
@@ -12,7 +12,7 @@ resource "azurerm_subnet" "apim" {
 }
 
 resource "azurerm_network_security_group" "apim" {
```

Comment:
Isn't this available in the [module](https://github.com/pagopa/dx/blob/2e0a35f3a2c23218fb85c8e9b1b9d19d345c5870/infra/modules/azure_api_management/network.tf#L34)?